build.assets/Dockerfile

```diff
@@ -48,7 +53,9 @@ RUN apt-get update -y --fix-missing && \
         net-tools \
         openssh-client \
         osslsigncode \
-        python-pip \
+        python3-pip \
```

Comment:
Are you sure it's a good idea to upgrade to Python 3? I'm not saying it's not, I'm just not sure all that it's used for in the first place and it seems like a major change.

Reply:
@ibeckermayer From what I know Python in this image is used only to run `yamllint` which doesn't even support python2 anymore https://github.com/adrienverge/yamllint/commit/a3fc64d13407869607c07096d8591bdc17ff645f. Teleport doesn't use Python, so application/tests are not affected by the change. Python2 is also EOL, so we shouldn't be used it anyways.
I think that the only reason why this image is using python2 is because it's default in Ubuntu 18.04.